.github/workflows/build-service.yml

```diff
@@ -10,55 +10,66 @@ on:
       - 'power-pay-backend/**'
       - '.github/workflows/build-service.yml'
 
-  pull_request:
-    branches:
-      - '**'
-    paths:
-      - 'power-pay-backend/**'
-      - '.github/workflows/build-service.yml'
-
 env:
   REGISTRY: ghcr.io
   IMAGE_NAME: ${{ github.repository }}-backend
 
 
 jobs:
   build:
-    if: github.event_name == 'pull_request'
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: power-pay-backend
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Clone the project
+        uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
           java-version: '17'
       - name: Maven build
-        run: cd power-pay-backend && mvn clean package -q
-      
+        run: mvn clean package -q
```

Comment:
here i can see mvn has been added in to the workflows file as required.

Reply:
Yes, because the working dir was specified